src/manifest.json

```
@@ -0,0 +1,22 @@
+{
+  "short_name": "quiver",
+  "name": "quiver",
+  "description": "a modern commutative diagram editor",
```

```suggestion
  "description": "A modern commutative diagram editor.",
```
I think this is the usual convention for `description`.